spec/features/course/assessment/question/multiple_response_management_spec.rb

```diff
@@ -41,8 +46,25 @@
         expect(current_path).to eq(course_assessment_path(course, assessment))
         expect(mrq.reload.maximum_grade).to eq(maximum_grade)
 
-        pending 'JavaScript support'
-        fail 'Add multiple options'
+        visit edit_path
+
+        options.each_with_index do |option, i|
+          click_link I18n.t('course.assessment.question.multiple_responses.form.add_option')
+          within all('.edit_question_multiple_response '\
+            "tr.question_multiple_response_option")[i] do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.